SU-HSDO/suhumsci
# HSD8-442 Disables sensitive permissions from UI

# READY FOR REVIEW

# Summary
- Disables checkbox in UI for any permission that is deemed to need "Restrict Access"

# Needed By (Date)
- 11/7

# Urgency
- medium

# Steps to Test
1. checkout this branch
1. clear caches
1. attempt to edit the permissions
1. validate checkboxes are disabled for all permissions with the description "Warning: Give to trusted roles only; this permission has security implications."
1. Review list of permissions if there might be anything that should be disabled.


# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
